src/test/GradleTestDevModeActions.ts

```diff
@@ -184,6 +185,24 @@ it('start gradle with history from liberty dashboard', async () => {
     
 }).timeout(350000);
 
+
```

Comment:
Instead of checking for tab count > -1, can we close all the other tabs and open only the reports and check the tab header title?
```suggestion
it('View test report for gradle project', async () => {

  // Close all the tabs in editor.
  await new EditorView().closeAllEditors();
  utils.delay(5000);

  if ((process.platform === 'darwin')) {
    //Function call to enter corresponding command in the command prompt to display test report for gradle project in mac
    await viewTestReportForMac();
  } else {
    await utils.launchDashboardAction(item, constants.GRADLE_TR_DASHABOARD_ACTION, constants.GRADLE_TR_DASHABOARD_MAC_ACTION);
  }
  tabs = await new EditorView().getOpenEditorTitles();
  await utils.delay(1000);
  console.log("Tabs opened: " + tabs.indexOf(constants.GRADLE_TEST_REPORT_TITLE));
  expect (tabs[0], "Gradle test report not found").to.equal(constants.GRADLE_TEST_REPORT_TITLE);

}).timeout(60000);
```